tests/browser_test.go

```diff
@@ -195,8 +195,10 @@ func TestBrowserVersion(t *testing.T) {
 	t.Parallel()
 
 	const re = `^\d+\.\d+\.\d+\.\d+$`
-	r, _ := regexp.Compile(re)
-	ver := newTestBrowser(t).Version()
+	r, err := regexp.Compile(re) //nolint:gocritic
```

Comment:
gocritic:

> regexpMust: for const patterns like re, use regexp.MustCompile

I disagree because panicking in tests produces a hard-to-diagnose goroutine stack trace. Handling the error provides concise information about the error's cause.